framework/Source/Kirakira.swift

```diff
@@ -103,6 +103,9 @@ public class Kirakira: OperationGroup {
     public var centerSaturation: Float = 0.3 {
         didSet { updateSparkleSaturation() }
     }
+    public var faceMaskImage: UIImage? {
+        didSet { sparklesEffect.faceMaskImage = faceMaskImage }
+    }
```

Comment:
This probably doesn’t belong in this PR.

Reply:
Sorry, it was caused by I chose the wrong base branch this morning 😓